packages/owl-bot/src/config-files.ts

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import Ajv from 'ajv';
+import owlBotYamlSchema from './owl-bot-yaml-schema.json';
```

Comment:
I believe the fact that we `import` it will ensure it's pulled into the build folder.